protocol-designer/src/components/organisms/SlotDetailModal/index.tsx

```diff
@@ -135,9 +135,13 @@ export const SlotDetailModal = (
       <Box
         backgroundColor={COLORS.grey10}
         padding={SPACING.spacing16}
-        height="28rem"
+        height={'28rem'}
       >
-        <Flex flexDirection={DIRECTION_ROW} gridGap={SPACING.spacing24}>
+        <Flex
+          flexDirection={DIRECTION_ROW}
+          gridGap={SPACING.spacing24}
+          height={'28rem'}
```

Comment:
same here, we like to follow the pattern of only using `{}` if its a const or `""` if its a string
```suggestion
          height="28rem"
```

Reply:
also why do we need to define the height of this when there is also the height defined in the box around it?